manifests/ppa.pp

```diff
@@ -5,6 +5,8 @@
   $options        = $::apt::ppa_options,
   $package_name   = $::apt::ppa_package,
   $package_manage = false,
+  $proxy_host     = undef,
+  $proxy_port     = 8080,
```

Comment:
Mmm, I've started disliking having multiple variables that are an obvious pair/entity as multiple switches on the class and instead have taken to doing something like this:

```puppet
class apt::params {
  $proxy = { 'host': undef, 'port': 8080 }
}

define apt::ppa (
  $proxy = {}
)  {
  $_proxy = merge($apt::proxy, $proxy)
}
```

This allows the user to still only specify `proxy => { 'host': '127.0.0.1' }` and get the port for free.

What do you think?